content/pages/nuttx/getting-started/index.md

```diff
@@ -0,0 +1,279 @@
+
+###### **ESP32**:
+
+Create a directory to install the toolchain. It can be on your home directory:
+```bash
+mkdir -p ~/xtensa-esp32-elf-gcc
+```
+
+Download and extract the toolchain on the new directory:
+```bash
+cd ~/xtensa-esp32-elf-gcc
+wget https://github.com/espressif/crosstool-NG/releases/download/esp-12.2.0_20230208/xtensa-esp32-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+tar -xf xtensa-esp32-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+```
+
+Once extracted, add the location of the binaries to the user’s PATH and reload the terminal:
+
+```bash
+echo "export PATH=$HOME/xtensa-esp32-elf-gcc/xtensa-esp32-elf/bin:\$PATH" >> ~/.bashrc
+```
+
+###### **ESP32-S2**:
+
+Create a directory to install the toolchain. It can be on your home directory:
+```bash
+mkdir -p ~/xtensa-esp32s2-elf-gcc
+```
+
+Download and extract the toolchain on the new directory:
+```bash
+cd ~/xtensa-esp32s2-elf-gcc
+wget https://github.com/espressif/crosstool-NG/releases/download/esp-12.2.0_20230208/xtensa-esp32s2-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+tar -xf xtensa-esp32s2-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+```
+
+Once extracted, add the location of the binaries to the user’s PATH and reload the terminal:
+```bash
+echo "export PATH=$HOME/xtensa-esp32s2-elf-gcc/xtensa-esp32s2-elf/bin:\$PATH" >> ~/.bashrc
+```
+
+###### **ESP32-S3**:
+
+Create a directory to install the toolchain. It can be on your home directory:
+```bash
+mkdir -p ~/xtensa-esp32s3-elf-gcc
+```
+
+Download and extract the toolchain on the new directory:
+```bash
+cd ~/xtensa-esp32s3-elf-gcc
+wget https://github.com/espressif/crosstool-NG/releases/download/esp-12.2.0_20230208/xtensa-esp32s3-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+tar -xf xtensa-esp32s3-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+```
+
+Once extracted, add the location of the binaries to the user’s PATH and reload the terminal:
+
+```bash
+echo "export PATH=$HOME/xtensa-esp32s3-elf-gcc/xtensa-esp32s3-elf/bin:\$PATH" >> ~/.bashrc
+```
+
+##### **RISC-V Devices (ESP32-C3, ESP32-C6, ESP32-H2)**
+
+All RISC-V devices use the same toolchain:
+
+Create a directory to install the toolchain. It can be on your home directory:
+```bash
+mkdir -p ~/riscv-none-elf-gcc
+```
+
+Download and extract the toolchain on the new directory:
+```bash
+cd ~/riscv-none-elf-gcc
+wget https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download/v13.2.0-2/xpack-riscv-none-elf-gcc-13.2.0-2-linux-x64.tar.gz
+tar -zxf xpack-riscv-none-elf-gcc-13.2.0-2-linux-x64.tar.gz
+```
+
+Once extracted, add the location of the binaries to the user’s PATH and reload the terminal:
+
+```bash
+echo "export PATH=$HOME/riscv-none-elf-gcc/xpack-riscv-none-elf-gcc-13.2.0-2/bin:\$PATH" >> ~/.bashrc
+```
+
+{{< alert >}}
+The toolchains will only be available after logging out and logging in from your user session or after a restart.
```

Comment:
This is not completely true, user does not need to logout or even reboot machine (we're in Linux world, not in Windows ;-) ), it's sufficient to source bashrc or just export the path.

Reply:
Agreed, I just tried to make it easier for beginners. I made it clearer in https://github.com/espressif/developer-portal/pull/45/files#diff-024a8213ef540f9f04acbb2d49638d2ba31f7dce21b9c0e8e0f613238bbac90fR121-R123